build/tasks/build.js

```diff
@@ -64,7 +64,11 @@ module.exports = function( grunt ) {
 		// Convert var modules
 		if ( /.\/var\//.test( path.replace( process.cwd(), "" ) ) ) {
 			contents = contents
-				.replace( /define\([\w\W]*?return/, "var " + ( /var\/([\w-]+)/.exec( name )[ 1 ] ) + " =" )
+				.replace(
+					/define\([\w\W]*?return/, "var " +
+					( /var\/([\w-]+)/.exec( name )[ 1 ] ) +
```

Comment:
Not very important but the indentation here seems weird to me, I'd expect `"var " +` to see in this line and the continuations indented by one more tab.